state/factory/statedb.go

```diff
@@ -318,10 +318,11 @@ func (sdb *stateDB) NewBlockBuilder(
 	ap actpool.ActPool,
 	sign func(action.Envelope) (action.SealedEnvelope, error),
 ) (*block.Builder, error) {
-	sdb.mutex.Lock()
 	ctx = protocol.WithRegistry(ctx, sdb.registry)
-	ws, err := sdb.newWorkingSet(ctx, sdb.currentChainHeight+1)
-	sdb.mutex.Unlock()
+	sdb.mutex.RLock()
+	h := sdb.currentChainHeight
```

Comment:
Replacing single char var `h` with longer var like `chainHeight` will make code editor easy to locate the var

Reply:
fixed